server/src/action/user/get.js

```diff
@@ -0,0 +1,19 @@
+const AbstractAction = require('../abstract')
+const { param } = require('express-validator')
+
+class GetUser extends AbstractAction {
+  get validate () {
+    return [
+      param('id')
+        .exists({ checkNull: true }).withMessage('required field')
+        .isInt().withMessage('must be an integer')
+        .toInt()
+    ]
+  }
+
+  async handler (req, res, next) {
+    return this.app.controller.user.get({ id: req.params.id })
```

Comment:
Unless i'm missing something, `app.controller.user.get` fetches the database, returning the whole user as it is stored. This would leak the user's password.

The `delete user.password` you did in the PR doesn't affect this call, it is unrelated, right?